sleap/nn/inference.py

```diff
@@ -121,6 +172,7 @@ def report_period(self) -> float:
     def from_model_paths(
         cls,
         model_paths: Union[str, List[str]],
```

Comment:
I decided to move the `model_name` to the new function `from_trained_models` because it could be cleaner that way separating pre-trained models from others that are not.

Reply:
The problem there is that we use the `Predictor.from_model_paths` to load all our `Predictor` subclasses whereas we define `Predictor.from_trained_models` as an [abstract method](https://realpython.com/python-interface/#using-abstract-method-declaration) (leaving it up to the subclasses to define in their own unique way).

Reply:
Good idea. I will try to modify `Predictor.from_model_paths` instead.